src/Xenial.Tasty/Tasty.Scope.cs

```diff
@@ -270,6 +274,31 @@ public async Task<int> Run(string[] args)
                 catch (IOException) { /* Handle is invalid */}
             }
 
+            if (ListenForInteractive)
+            {
+                var isInteractive = Environment.GetEnvironmentVariable("TASTY_INTERACTIVE");
+                if (!string.IsNullOrEmpty(isInteractive))
+                {
+                    if (bool.TryParse(isInteractive, out var result))
+                    {
+                        if (result)
+                        {
+                            Console.WriteLine("INTERACTIVE MODE ACTIVATED");
+                            var connectionType = Environment.GetEnvironmentVariable("TASTY_INTERACTIVE_CON_TYPE");
+                            Console.WriteLine(connectionType);
+                            if (connectionType == "NamedPipes")
+                            {
+                                var connectionId = Environment.GetEnvironmentVariable("TASTY_INTERACTIVE_CON_ID");
+                                Console.WriteLine(connectionId);
+                                var stream = new NamedPipeClientStream(".", connectionId, PipeDirection.InOut, PipeOptions.Asynchronous);
+                                await stream.ConnectAsync();
```

Comment:
Consider using a CancelationTokenSource with timeout here, could block forever if server stream isn't established ever

Reply:
Good point, currently I'm playing with the design and protocol, so it's a little bit early for a review yet though.